components/molecule/rating/src/index.scss

```diff
@@ -0,0 +1,78 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~react-rating-svg/lib/scss/react-rating-svg';
+
+$fz-molecule-rating-small: $fz-s !default;
+$fz-molecule-rating-medium: $fz-m !default;
+$fz-molecule-rating-large: $fz-l !default;
+
+$c-molecule-rating: #5f5f5f !default;
+$c-molecule-rating-label-hover: #5398c1 !default;
+$bg-molecule-rating-hover: #ecf4f8 !default;
+
+$base-class: '.sui-MoleculeRating';
+$class-star: '#{$base-class}-Star';
+$class-label: '#{$base-class}-label';
+
+#{$base-class} {
+  color: $c-molecule-rating;
+  display: flex;
+  align-items: center;
+
+  &-containerStars {
+    display: flex;
+  }
+
+  #{$class-label} {
+    display: flex;
+    padding: $p-m;
+    margin-left: $m-m;
```

Comment:
<p> tag usually has a margin top and bottom by default. Could it be `$m-molecule-rating-label: 0 0 0 $m-m !default` in order to avoid this and make it parametrizable by project?

Reply:
done!